components/GoalsCard/GoalsCard.js

```diff
@@ -0,0 +1,20 @@
+import { StyledItem, StyledList, StyledGoalCard, StyledText } from "@/styles";
+
+export default function GoalsCard({ goals }) {
+  return (
+
+      <StyledList>
+        {goals.map(({ id, goalName, savedAmount, goalAmount }) => (
+          <StyledItem key={id}>
+            <div>
+              <StyledGoalCard>{goalName}</StyledGoalCard> <br></br>{" "}

```

Comment:
you could use margin /  padding instead of br then you also wouldn't need the {"  "}. and a semantic element instead of the styledgoalcard div

Reply:
done ty